Formula/ibazel.rb

```diff
@@ -0,0 +1,30 @@
+class Ibazel < Formula
+  desc "Tools for building Bazel targets when source files change"
+  homepage "https://github.com/bazelbuild/bazel-watcher"
+  url "https://github.com/bazelbuild/bazel-watcher/archive/refs/tags/v0.16.2.tar.gz"
+  sha256 "a927520e7ab3a1fb749043e543c57bb211666cd627d053fbe0e8245730beee75"
+  license "Apache-2.0"
+
+  depends_on "bazel" => :build
+
+  def install
+    # Bazel clears environment variables other than PATH during build, which
+    # breaks Homebrew shim scripts. We don't see this issue on macOS since
+    # the build uses a Bazel-specific wrapper for clang rather than the shim;
+    # specifically, it uses `external/local_config_cc/wrapped_clang`.
+    #
+    # The workaround here is to disable the Linux shim for C/C++ compilers.
+    # Remove this when a way to retain HOMEBREW_* variables is found.
+    ENV["CC"] = "/usr/bin/cc"
+    ENV["CXX"] = "/usr/bin/c++"
```

Comment:
I don't think we want to add new formulae that don't use the shims.

Reply:
Updated. I believe it's not possible to use the clang shims under Bazel as things stand, but the xcode toolchain does work.